.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "EditorConfig.EditorConfig",
+    "esbenp.prettier-vscode",
```

Comment:
JSONなので末尾のcommaは削除した方が良いと思いました。
他のファイルも同様です。

Reply:
全てのextensions.json、settings.jsonファイルの末尾のカンマを削除しました。

Reply:
FYI

公式情報は見つけられてないですが、VSCode設定の`.json` は jsonc らしいです。

- [JSONとJSONCとJSON5の違い - mrsekut-p](https://scrapbox.io/mrsekut-p/JSON%E3%81%A8JSONC%E3%81%A8JSON5%E3%81%AE%E9%81%95%E3%81%84)
- [\[settings\] Use JSON5 instead of JSONC for configuration files · Issue #100688 · microsoft/vscode](https://togithub.com/microsoft/vscode/issues/100688)
  - > Currently configuration files are in JSONC format

----

jsonにする場合、

`"source.fixAll.eslint":true,` の部分が残っているようです？

```
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint":true,
  },
  "eslint.workingDirectories": [
```

Reply:
> jsonにする場合、"source.fixAll.eslint":true, の部分が残っているようです？

対応commit: https://github.com/ws-4020/mobile-app-crib-notes/pull/1249/commits/07a3979078d2c0b6233214c6bfe920a36b4f4f91